src/Orchard.Web/Modules/Orchard.OpenId/Startup.cs

```diff
@@ -27,13 +28,13 @@ public class Startup : StartupBase
 
         public Startup(
             ShellSettings shellSettings,
-            IAppDataFolder appDataFolder,
+            IHostingEnvironment environment,
```

Comment:
@sebastienros not sure this is the right fix, so please correct me if I'm wrong :smile: